### ISSUE TITLE
How long do you run this code(preprocess_data.m)?

### ISSUE THREAD

**Issue by Taikangsheng** [nachiket92/conv-social-pooling#12]
The code is running for more than 18 hours in MATLAB 2018b and the result haven't come out yet.  The state is always "connected to 6 workers". How long do you run this code(preprocess_data.m)?

**Comment by nachiket92** [comment]
Yeah, it's a little slow. It takes about a day to run with 6 workers.

**Comment by beyond96** [comment]
出错 pre_process (line 58)
poolobj = parpool(6);

You requested a minimum of 6 workers, but the cluster "local" has the NumWorkers property set to allow a maximum of 4 workers. To run a communicating job on more workers than this (up to a maximum of 512 for the Local cluster), increase the value of the NumWorkers property for the cluster. The default value of NumWorkers for a Local cluster is the number of cores on the local machine.

How can I solve this question?

**Comment by beyond96** [comment]
How long will it take to run with 4 workers?

**Comment by varunjammula** [comment]
How do you set the workers for a Matlab script?

**Comment by JustMrZuo** [comment]
I used a regular laptop released in 2021 and the total time taken was about half an hour.
